src/GameLoveLetter/Cards/BaronCard.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace GameLoveLetter.Cards
+{
+	public class BaronCard : ICard<BaronCardData>
+	{
+		public int Strength { get; set; }
+		public BaronCardData Data { get; set; }
+
+		public BaronCard()
+		{
+			Strength = 3;
```

Comment:
init from declaration like this `public int Strength { get; set;} = 3;`

Reply:
Okay